### PR TITLE
fix: postage stamp price and TTL calculation

### PR DESCRIPTION
**Pull request by vojtechsimetka**

### CODE REVIEW

**Comment by bee-worker** [comment]
## 🐝 PR preview in Swarm

**Preview URL:** https://bah5acgzatcmzkj6pjmwrvxmzg53pucnmyscbxqtvhuhu3a34tahvsqukkr2a.bzz.link
**Swarm Hash:** `98999527cf4b2d1add993776fa09acc4841bc2753d0f4d837c980f59428a5474`
**Commit Hash:** 00d1f633d7cb914b0bd376c0133641678c2db699
**Commit Message:** chore: removed logs and fixed linter issues

File: src/utils/index.ts [hunk 1]
```diff
@@ -159,38 +159,21 @@ export function secondsToTimeString(seconds: number): string {
...
-export function calculateStampPrice(depth: number, amount: number, currentPrice: NumberString): number {
-  const price = parseInt(currentPrice, 10)
+  // See https://github.com/ethersphere/bee/blob/66f079930d739182c4c79eb6008784afeeba1096/pkg/debugapi/postage.go#L410-L413
+  return (amount * blockTime) / pricePerBlock
+}
 
-  return (amount * 2 ** (depth - 16) * price) / 1e16
+export function calculateStampPrice(depth: number, amount: bigint): Token {
+  // See https://github.com/ethersphere/bee/blob/66f079930d739182c4c79eb6008784afeeba1096/pkg/debugapi/postage.go#L410-L413
+  return new Token(amount * BigInt(2 ** depth)) // FIXME: the 2 ** depth should be performed on bigint already
```
Comment (agazso) [inline src/utils/index.ts:176]:
```suggestion
  return new Token(amount * (2n ** BigInt(depth)))
```

Comment (vojtechsimetka) [reply src/utils/index.ts:176]: yeah but our compilation target does not support power on BigInt :(

**Review by Cafe137 — APPROVED** [review]

**Comment by Cafe137** [comment]
Nice, I like how you improved the `toSignificantDigits` 🙂

**Pull request merged**